dockerfiles/notebook/Dockerfile

```
@@ -2,10 +2,17 @@ FROM jupyter/datascience-notebook:python-3.10.6
 #FROM jupyter/datascience-notebook:d53a302fbcd0
 USER root
```

```suggestion
USER root
ENV DEBCONF_NOWARNINGS yes
```

https://github.com/The-Japan-DataScientist-Society/100knocks-preprocess/pull/147 で削除された差分には上記も含まれていますので、復活していただけるとありがたいです。

---

復活しました！